src/components/Header.js

```diff
@@ -0,0 +1,225 @@
+/* eslint-disable max-len */
+import React from 'react';
+import styled from 'styled-components';
+import classNames from 'classnames';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faMagnifyingGlass, faBars } from '@fortawesome/free-solid-svg-icons';
+import useSticky from './useSticky';
```

Comment:
What is this Sticky for? I want to know more about it - hopefully I'll learn more about it in the future :) 
I guess there is an npm-command for it too, will Google it tomorrow :)